libs/libcommon/src/libcommon/config.py

```diff
@@ -270,6 +272,16 @@ class ProcessingGraphConfig:
                 "requires": ["split-first-rows-from-streaming"],
                 "job_runner_version": PROCESSING_STEP_SPLIT_OPT_IN_OUT_URLS_SCAN_VERSION,
             },
+            "config-opt-in-out-urls-scan": {
+                "input_type": "config",
+                "requires": ["split-opt-in-out-urls-scan"],
```

Comment:
I think it also requires the list of split names, right?

Reply:
Yes I added it here https://github.com/huggingface/datasets-server/blob/config-dataset-spawning-level/libs/libcommon/src/libcommon/config.py#L283